Sources/TuistCache/Cache/Cache.swift

```diff
@@ -11,8 +11,8 @@ public final class Cache: CacheStoring {
 
     // MARK: - Init
 
-    public convenience init() {
-        self.init(storages: [CacheLocalStorage()])
+    public convenience init(cloudClient: CloudClienting) {
```

Comment:
I'd add a convenience initializer to `CacheRemoteStorage` that takes no arguments and then remove the `cloudClient` argument here. Unless there's a reason for dependency injection through the constructor at this level?